cpu/ppc64_cpu_test.py

```diff
@@ -147,6 +152,14 @@ def dscr(self):
         command2 = "cat /sys/devices/system/cpu/dscr_default"
         self.equality_check("DSCR", command1, command2)
 
+    def smt_loop(self):
+        """
+        Tests smt on/off in a loop
+        """
+        for i in range(1, 100):
+            process.system_output("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
```

Comment:
@sacsant can we follow an approach similar to https://github.com/avocado-framework-tests/avocado-misc-tests/pull/403/files